core/src/runtime/World.ts

```diff
@@ -170,6 +170,7 @@ export const World = ({ commands, games, renderer, runtimeMode }: WorldProps): W
       // run system onTick
       Object.values(world.systems).forEach((system) => {
         if (!isRollback || (isRollback && !system.skipOnRollback)) {
+          if (!world.systems[system.id]) return;
```

Comment:
checking if the system is still registered in the `World`. need to do this because `ActionSystem` invokes the game change mid-onTick